doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -264,79 +266,102 @@ def draw_thresh_map(
 
     def build_target(
         self,
-        target: List[np.ndarray],
-        output_shape: Tuple[int, int, int],
+        target: Union[List[Dict[str, np.ndarray]], List[np.ndarray]],
+        output_shape: Tuple[int, int, int, int],
```

Comment:
There is a new dimension here. Is it for the number of target ?

Reply:
before they didn't take also the class shape because they assumed it was 1 now i take it also